src/en/news/blog/2022/dashboard-rbd-scaling/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Dashboard scability of RBD images"
+date: "2022-09-01"
+author: "Pere Diaz Bou"
+image: "images/image.jpg"
+tags:
```

Comment:
Dashboard tag? Scalability? Have a look at the existing tags and add then ones this post would be meaningful to.